src/server/components/dashboard/listsItems/item/update/actionHandlers/requestChanges.ts

```diff
@@ -80,7 +81,7 @@ async function handleListItemRequestChanges(
         where: { id: listItem.id },
         data: {
           status,
-          isPublished: false,
```

Comment:
I think in this case we can just omit this line if we want to keep it the same

Reply:
yep, makes sense.  Just pushed up that change.